.github/workflows/release.yaml

```
@@ -0,0 +1,112 @@
+name: Promote Operator Release
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: 'Name for the tag of the release.'
+        required: true
+      quay_registry:
+        description: 'Quay registry to push to.'
+        default: 'quay.io/ansible'
+
+jobs:
+  promote-staged-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set TAG_NAME for workflow_dispatch event
+        if: ${{ github.event_name == 'workflow_dispatch' }}
+        run: |
+          echo "TAG_NAME=${{ github.event.inputs.tag_name }}" >> $GITHUB_ENV
+          echo "QUAY_REGISTRY=${{ github.event.inputs.quay_registry }}" >> $GITHUB_ENV
+
+      - name: Set TAG_NAME for release event
+        if: ${{ github.event_name == 'release' }}
+        run: |
+          echo "TAG_NAME=${{ github.event.release.tag_name }}" >> $GITHUB_ENV
+
+      - uses: actions/checkout@v3
+        with:
+          ref: 'main'
+          depth: 0
+          path: eda-server-operator
+
+      - name: Log into registry ghcr.io
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d    # v3.0.0
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+
+      - name: Log into registry quay.io
+        uses: docker/login-action@343f7c4344506bcbf9b4de18042ae17996df046d    # v3.0.0
+        with:
+          registry: ${{ github.event.inputs.quay_registry }}
+          username: ${{ secrets.QUAY_USER }}
+          password: ${{ secrets.QUAY_TOKEN }}
+
+      - name: Pull Stored Image and Publish to ${{ github.event.inputs.quay_registry }}/eda-server-operator:${TAG_NAME}
+        run: |
+          docker buildx imagetools create \
+            ghcr.io/${{ github.repository }}:${TAG_NAME} \
+            --tag ${{ github.event.inputs.quay_registry }}/eda-server-operator:${TAG_NAME}
+        working-directory: eda-server-operator
+
+      - name: Pull Stored Image and Publish to ${{ github.event.inputs.quay_registry }}/eda-server-operator:latest
+        run: |
+          docker buildx imagetools create \
+            ghcr.io/${{ github.repository }}:${TAG_NAME} \
+            --tag ${{ github.event.inputs.quay_registry }}/eda-server-operator:latest
+        working-directory: eda-server-operator
+
+      - name: Build Bundle Image
+        run: |
+          make bundle IMG=eda-server-operator:${{ github.event.inputs.tag_name }} VERSION=${{ github.event.inputs.tag_name }}
+          make bundle-build BUNDLE_IMG=eda-server-operator-bundle:${{ github.event.inputs.tag_name }}
+          docker tag eda-server-operator-bundle:${{ github.event.inputs.tag_name }} eda-server-operator-bundle:latest
+        working-directory: eda-server-operator
+
+      - name: Push Bundle Image
+        uses: redhat-actions/push-to-registry@v2.1.1
```

Any reason not use the latest version here ? 2.1.1 is 3 years old
```suggestion
        uses: redhat-actions/push-to-registry@v2.7.1
```